modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -41,6 +42,110 @@
+
+User Key encryption has been already deprecated in core versions later than 10.7. For this reason, the command line interface and web UI to enable User Key encryption is no longer available. If you are operating an ownCloud installation with User Key encryption enabled, please get in contact with support@owncloud.com to plan a migration to Master Key encryption. https://github.com/owncloud/encryption/pull/389[encryption#389]
+
+=== User Key encryption is no longer auto-enabled
+
+Executing the "encryption:encrypt-all" occ command will no longer auto-enable User Key encryption. https://github.com/owncloud/core/pull/40702[#40702]
+
+=== Added occ command to remove obsolete storages
+
+Metadata coming from storages are stored in the DB. Previously, when a storage has been removed from ownCloud that metadata remained in the DB. The new occ command allows to remove that metadata stored, reducing the amount of space used by the DB as well as slightly improving the performance since there will be less entries. https://github.com/owncloud/core/pull/40779[#40779]
+
+=== Enforce 2-factor authentication
+
+2-factor authentication can be enforced now. The feature requires at least an app implementing the 2-factor, otherwise no enforcement will be done. If the 2-factor authentication is enforced, all users will be required to use a 2-factor authentication app. Some specific groups selected by the admin can be excluded to let those users bypass the 2-factor authentication. https://github.com/owncloud/core/pull/40830[#40830]
+
+=== Setting up federation sharing over the WebUI is again possible
+
+After update of guzzle, it was no longer possible to request format of response to be json when adding in query parameter with the consequence that setting up federation sharing over the WebUI was not possible. One of the "OCSAuthAPIController" federated instances was receiving requests without a hint that JSON needs to be used, and returned XML. On the other hand, "OCSAuthAPIController" expects only JSON for exchange, and thus failed to parse the message. Now the exchange is correctly done. https://github.com/owncloud/core/pull/40815[#40815]
+
+=== Add commands to handle the trusted servers from command line
+
+New occ commands have been added to handle the trusted servers for federation from command line. These commands will allow the admin to add list and remove trusted servers. https://github.com/owncloud/core/pull/40796[#40796]
+
+=== Skip share_folder for guest users
+
+In https://github.com/owncloud/core/pull/40378 we have fixed the case of (not) moving the share target when the backend storage becomes temporary unavailable but we had the collateral effect that guests did not see anymore their received shares as we were forcing the creation of the target which failed for them as their storage is read-only. We now skip the share_folder config.php option for guests and default to root. https://github.com/owncloud/core/pull/40864[#40864]
```

Comment:
```suggestion
In https://github.com/owncloud/core/pull/40378 we have fixed the case of (not) moving the share target when the backend storage becomes temporarily unavailable but we had the collateral effect that guests did not see their received shares any more as we were forcing the creation of the target which failed for them as their storage is read-only. We now skip the share_folder config.php option for guests and default to root. https://github.com/owncloud/core/pull/40864[#40864]
```

Maybe this also has too much detail for these kind of release notes?

Reply:
adjusted